docs/editor.md

```diff
@@ -49,7 +49,7 @@ Here's a quick description of each:
         "files.trimFinalNewlines": true,
         "files.insertFinalNewline": true,
         // Testing settings
-        "mochaExplorer.files": "test/unit/**/*.test.js",
+        "mochaExplorer.files": "test/unit/**/*.test.*",
```

Comment:
This updates the recommended workspace settings for VSCode so that mocha test explorer picks up typescript tests.